tests/executors/test_dask_executor.py

```diff
@@ -58,8 +57,8 @@ def assert_tasks_on_executor(self, executor):
         success_command = ['true', 'some_parameter']
         fail_command = ['false', 'some_parameter']
 
-        executor.execute_async(key='success', command=success_command)
-        executor.execute_async(key='fail', command=fail_command)
+        executor.execute_async(key='success', command_string=success_command)
+        executor.execute_async(key='fail', command_string=fail_command)
```

Comment:
Why did you change this? It is still `command`, isn't it? 🤔

Reply:
Left-over from a temp problem. Reason why some tests failed.